src/main/java/org/primefaces/extensions/component/timeago/TimeAgo.java

```diff
@@ -52,67 +52,6 @@
 
     private static final String DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ssZ";
 
-    private static final List<String> BUNDLED_LOCALES = Arrays.asList(
```

Comment:
This should have been removed in the previous fix where we get the translations from Javascript, so I included it in this PR.